docker/backend-dev/conf/Config2.pm

```diff
@@ -26,53 +26,71 @@ use Exporter    qw< import >;
 
 BEGIN
 {
-	use vars       qw(@ISA @EXPORT_OK %EXPORT_TAGS);
+	use vars       qw(@ISA @EXPORT @EXPORT_OK %EXPORT_TAGS);
+	require Exporter;
+	@ISA = qw(Exporter);
+	@EXPORT = qw();
 	@EXPORT_OK = qw(
 		$server_domain
 		@ssl_subdomains
+		$producers_platform
 		$data_root
 		$www_root
 		$geolite2_path
 		$mongodb
 		$mongodb_host
 		$mongodb_timeout_ms
 		$memd_servers
-		$facebook_app_id
-	    $facebook_app_secret
+		$google_cloud_vision_api_key
+		$crowdin_project_identifier
+		$crowdin_project_key
+		$postgres_user
+		$postgres_password
 		$robotoff_url
 		%server_options
 	);
 	%EXPORT_TAGS = (all => [@EXPORT_OK]);
 }
 use vars @EXPORT_OK ; # no 'my' keyword for these
+use strict;
+use utf8;
 
 # server constants
-$server_domain = "productopener.localhost";
+$server_domain = $ENV{PRODUCT_OPENER_DOMAIN};
 
-@ssl_subdomains = qw();
+@ssl_subdomains = split / /, $ENV{SSL_SUBDOMAINS};
+$producers_platform = $ENV{PRODUCERS_PLATFORM};
```

Comment:
Can we use this to replace the https://github.com/openfoodfacts/openfoodfacts-server/tree/main/docker/backend-pro-dev and update https://github.com/openfoodfacts/openfoodfacts-server/blob/main/docker/docker-compose.pro.dev.yml accordingly? @stephanegigandet

Reply:
Will make a separate PR for this, since adding env variables gives us a lot of freedom regarding having a unified Config2.pm / Dockerfile / docker-compose file, parameterized through env vars.